### ISSUE TITLE
App in chrome web store not found 404

### ISSUE THREAD

**Issue by bryanbocao** [binarylu/leetcode-ext#19]
![screen_shot_2017-04-03_at_2_05_42_am](https://cloud.githubusercontent.com/assets/14010288/24600686/f2f587a4-1812-11e7-8a40-f0601004afec.jpg)

**Comment by lkjhgfdsaa** [comment]
I agree with @BryanBo-Cao - I am neither able to see the locked questions, neither the company tags.  It does open up a modal, but then it is just blank.  Request you to kindly help in this, @binarylu I appreciate your help.

**Comment by lkjhgfdsaa** [comment]
@binarylu Could you please help us here?  Thanks! 👍
